com.dynamo.cr/com.dynamo.cr.bob/build.gradle

```diff
@@ -183,23 +183,23 @@ task compileBobLight(dependsOn: [getGitRevision, compileSrc, createLuaJitShareZi
                 into "${project.bobDir}/libexec/$platform"
             }
         }
-
+        def defaultTarName = 'luajit-2.1.0-a4f56a4'
```

Comment:
Since I can't rebuild `luajit` for `arm64-linux` I kept old one, gradle script modified to use different luajit versions